apps/backend/src/lambdas/parser.ts

```diff
@@ -65,15 +83,56 @@ export const handler = async (event?: unknown, _context?: Context) => {
 
       // TODO [CCFPCM-318] Excluded LABOUR2 files that are DDF, needs implementation
       const finalParseList = parseList.filter(
-        (filename) => !filename?.includes('LABOUR2')
+        (filename) => !filename?.includes('LABOUR')
       );
+      appLogger.log('Creating daily upload for today if needed');
+
+      axios.request({
+        method: 'post',
+        maxBodyLength: 100000,
+        url: `${API_URL}/v1/parse/daily-upload`,
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        data: {
+          date: new Date(),
+        },
+      });
 
       // Parse & Save only files that have not been parsed before
       for (const filename of finalParseList) {
         appLogger.log(`Parsing ${filename}..`);
         const event = { eventType: 'all', filename: filename };
         await processEvent(event);
       }
+
+      let alertsSentResponse = await axios.request({
```

Comment:
Should we create an axios instance and pass in params to make this configurable? Just to avoid duplication?